Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.EntityFrameworkCore;
 using Promact.Trappist.Web.Models;
```

Comment:
why you have removed namespace? i asked you to update it as per feature, not to remove it.